src/app/routes/groupRoutes.js

```diff
@@ -1,6 +1,7 @@
 const proxyUtils = require('../proxy/proxyUtils.js')
 const envHelper = require('../helpers/environmentVariablesHelper.js')
-const learnerURL = envHelper.LEARNER_URL
+const utils = require('../helpers/utils.js');
+const learnerURL  = utils.defaultHost(utils.envVariables.LEARNER_URL);
```

Comment:
Please add checks
const learnerURL = utils?.defaultHost(utils?.envVariables?.LEARNER_URL);

Reply:
added safety checks for all places with utils.defaultHost